pkg/sql/logictest/testdata/logic_test/create_table

```diff
@@ -169,9 +171,10 @@ like_constraints  CREATE TABLE public.like_constraints (
                   h INT8 NULL,
                   j JSONB NULL,
                   k INT8 NULL,
+                  t TIMESTAMPTZ NULL ON UPDATE current_timestamp():::TIMESTAMPTZ,
```

Comment:
hm this table was made with `INCLUDING CONSTRAINTS` so it shouldn't have `ON UPDATE` right? (similar to how it doesn't have the `DEFAULT` on `b`